src/handlers/check_run/completed.js

```diff
@@ -11,19 +11,24 @@ import * as circleci from '../../circleci';
 import github from '../../github';
 import Logger from '../../logger';
 import { replPreview } from '../../messages';
-import type { StatusPayload } from './types';
+import type { CheckRunPayload } from './types';
 
-const { log } = new Logger('status/success.js');
+const { log } = new Logger('check_run/completed.js');
 
-export default async function(payload: StatusPayload): Promise<void> {
+export default async function(payload: CheckRunPayload): Promise<void> {
   // We only care about CircleCI statuses
-  if (payload.context !== 'ci/circleci') {
-    log(`Ignoring payload ${payload.id} as it is not CircleCI: ${payload.context}`, 'verbose');
+  const { check_run } = payload;
+  if (check_run.name !== 'build-standalone') {
+    log(`Ignoring check run ${check_run.id} as it is not build-standalone checks: ${check_run.name}`, 'verbose');
     return;
   }
 
+  if (check_run.conclusion !== "success") {
+    log(`Ignoring check run ${payload.check_run.id} as it does not conclude successfully: ${check_run.conclusion}`, 'verbose');
```

Comment:
so we don't want to return here? I guess it's unlikely this would fail but if the bundle fails I guess it can fail right?

Reply:
Nice catch.